lib/spinner.js

```diff
@@ -0,0 +1,22 @@
+const helper = require('./utils/helper'),
```

Comment:
A semicolon should be present at the end of this import.

Reply:
The latest commit was done post rebasing.